tests/mesolve/test_batching.py

```diff
@@ -0,0 +1,72 @@
+import jax
+import pytest
+from jax import numpy as jnp
+
+import dynamiqs as dq
+
+
+@pytest.mark.parametrize('cartesian_batching', [True, False])
+def test_batching(cartesian_batching):
+    n = 8
+    nt = 11
+    nH = 3
+    nLs = 4 if cartesian_batching else nH
+    npsi0 = 5 if cartesian_batching else nH
+    nEs = 6
+
+    options = dq.options.Options(cartesian_batching=cartesian_batching)
+
+    # create random objects
+    k1, k2, k3, k4 = jax.random.split(jax.random.PRNGKey(42), 4)
+    H = dq.rand.herm(k1, (nH, n, n))
+    Ls = dq.rand.herm(k2, (nLs, 1, n, n))
```

Comment:
```suggestion
    Ls = dq.rand.herm(k2, (nLs, 2, n, n))
```
to make sure the sum over `jump_ops` happens correctly.